libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -259,7 +412,31 @@ async def create(
         assistant_id: Optional[str] = None,
         if_exists: Optional[OnConflictBehavior] = None,
     ) -> Assistant:
-        """Create a new assistant."""
+        """Create a new assistant.
+
+        Useful when graph is configurable and you want to create different assistants based on different configurations.
+
+        Args:
+            graph_id (str): The ID of the graph the assistant should use. The graph ID is normally set in your langgraph.json configuration.
+            config (Config, optional): Configuration to use for the graph. Defaults to None.
+            metadata (dict, optional): Metadata to add to assistant. Defaults to None.
+            assistant_id (str, optional): Assistant ID to use, will default to a random UUID if not provided.
+            if_exists (OnConflictBehavior, optional): How to handle duplicate creation. Defaults to None.
```

Comment:
can we document this better:

(1) it actually will default to `raise` under the hood
(2) `raise` = raise an error if there is a duplicate
(3) `do_nothing` = return existing assistant

Reply:
same for threads